lacommunaute/users/views.py

```diff
@@ -24,7 +24,7 @@
 from lacommunaute.utils.urls import clean_next_url
 
 
-logger = logging.getLogger(__name__)
+logger = getLogger("lacommunaute")
```

Comment:
Personnellement j'aime bien avoir des loggers différents pour retrouver plus facilement d'où vient un log.

Reply:
💯 d'accord avec toi. Mais J'ai cru noté que les logs n'étaient pas attrapées avec la conf dans `base.py` si je ne mettais pas un nom fixe. Certainement un défaut de conf de ma part.